docs/developer/getting-started/debugging.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[kibana-debugging]]
+=== Debugging {kib}
+
+* <<debugging-unit-tests>>
```

Comment:
To add some context, maybe change this to something like:

For information about how to debug unit tests, refer to <<debugging-unit-tests>>.